includes/discovery/sensors/temperature/avaya-vsp.inc.php

```diff
@@ -0,0 +1,71 @@
+/**
+ * avaya-vsp.inc.php
+ *
+ * LibreNMS Temperature Sensor Discovery module for Avaya VOSS(VSP Operating System Software)
+ *
+ * Copyright (c) 2017 Daniel Cox <danielcoxman@gmail.com>
+ *
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+ */
+     
+$high_limit = 55;
+$low_limit = 0;
+     
+// rcChasFanAmbientTemperature
+$tempsfan = snmp_walk($device, '.1.3.6.1.4.1.2272.1.4.7.1.1.3', '-Osqn');
```

Comment:
You should change your walks to use snmpwalk_cache_*, they provide the data in an array so you don't need to do explodes and things.